hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2935,7 +2935,7 @@
     <description>
       The base dir for HTTP Jetty server to extract contents. If this property
       is not configured, by default, Jetty will create a directory inside the
-      directory named by the java.io.tmpdir System property（/tmp by default）.
+      directory named by the ozone.metadata.dirs System property（/tmp/metadata by default）.
```

Comment:
ozone.metadata.dirs doesn't have default value. It's a must configured property in Ozone, also not a  "System property".

Reply:
Thanks for the review, I have made changes to it do take a look!